WORKSPACE

```diff
@@ -1,3 +1,7 @@
+PROXY = "eb8bfccc391e528fba316c4e081515eb18583e5c"
+
+PROXY_SHA = ""
```

Comment:
Yes there is no SHA for that particular commit. "" is the default, meaning no SHA.